### ISSUE TITLE
Searching Drug-Gene Interactions Status

### ISSUE THREAD

**Issue by ahwagner** [genome/dgi-db#16]
Searching Drug-Gene Interactions status window doesn't display in Safari browser.

**Comment by acoffman** [comment]
Are you referring to the modal pop-up that appears after you click "search"? Is the same true for search categories then?

**Comment by ahwagner** [comment]
Yes, I'm referring to the modal window pop-up after submitting a search. True for search categories as well.
